content/documentation/staging/tutorial/01-prerequisites/03-update-kiali.adoc

```diff
@@ -0,0 +1,29 @@
+---
+title: Update Kiali
+weight: 14
```

Comment:
On my machine I see the section "Access the Kiali UI" before "Update Kiali", which I guess isn't intended. I can see they have the same weight, perhaps that's the reason.

Reply:
Ouch, yes, thanks good catch !